Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/hitscan.yml

```diff
@@ -16,12 +16,12 @@
       - HideContextMenu
   - type: AnimationPlayer
 
+# starlight start
```

Comment:
nit: having a description of what's being modified in this block may help on future merges.

Reply:
done